### ISSUE TITLE
ESLint errors 'NavigationActions not found in 'react-navigation'' 

### ISSUE THREAD

**Issue by maribies** [eslint/eslint#12761]
**Tell us about your environment**

* **ESLint Version:** v6.5.1
* **Node Version:** v11.15.0
* **npm Version:** v6.11.3

**What parser (default, Babel-ESLint, etc.) are you using?**
Babel v7.6.2

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

software | version
-- | --
react-navigation | 4.0.10
react-native | 0.61.4
node | v11.15.0
npm or yarn | 1.16.0

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
yarn lint
```
I am trying to bump the React Navigation version in my project but I am facing some errors in ESLint.

**What did you expect to happen?**

The imports to be found.

**What actually happened? Please include the actual, raw output from ESLint.**

  1:10  error  NavigationActions not found in 'react-navigation'  import/named
  1:29  error  StackActions not found in 'react-navigation'       import/named
  80:10  error  createAppContainer not found in 'react-navigation'  import/named

I know the imports are found because the app doesn't crash and works as expected.

I first opened an issue with React Navigation,  and was told by the main maintainer it is an issue with "ESLint plugin that it doesn't recognize TypeScript definitions."  See the original on the react-navigation repo [issue](https://github.com/react-navigation/react-navigation/issues/6506).

Any suggestions?

Thanks.

**Comment by g-plane** [comment]
Hello, it seems that this problem is from eslint-plugin-import, not ESLint itself. You may consider opening issue at its repository.

**Comment by kaicataldo** [comment]
As @g-plane mentioned, this error is coming from [this plugin](https://github.com/benmosher/eslint-plugin-import) and will have to be solved there. Closing since this question has been answered, but feel free to visit our [Gitter chat](https://gitter.im/eslint/eslint) or to make a new issue if something else comes up!
